crates/oxc_minifier/src/ast_passes/statement_fusion.rs

```diff
@@ -312,14 +331,14 @@ mod test {
         fuse_same("a; { b; const otherVariable = 1; }");
 
         // enable_normalize();
-        test(
-            "function f(a) { if (COND) { a; { b; let a = 1; } } }",
-            "function f(a) { if (COND) { { a,b; let a$jscomp$1 = 1; } } }",
-        );
-        test(
-            "function f(a) { if (COND) { a; { b; let otherVariable = 1; } } }",
-            "function f(a) { if (COND) {  { a,b; let otherVariable = 1; } } }",
-        );
+        // test(
+        //     "function f(a) { if (COND) { a; { b; let a = 1; } } }",
+        //     "function f(a) { if (COND) { { a,b; let a$jscomp$1 = 1; } } }",
+        // );
+        // test(
+        //     "function f(a) { if (COND) { a; { b; let otherVariable = 1; } } }",
+        //     "function f(a) { if (COND) {  { a,b; let otherVariable = 1; } } }",
+        // );
```

Comment:
Closure checks whether `isASTNormalized` (here) https://github.com/google/closure-compiler/blob/081f46cb61c8852ec6a863db18e6a9d91693ce23/src/com/google/javascript/jscomp/StatementFusion.java#L92

oxc_minifier doesn't have this concept yet, so i guess its ok to leave these commented out for now?

Reply:
Yes. I don't even know what "normalized" ast means.